README.md

```diff
@@ -96,7 +97,7 @@ the `--user` and --group` flags to `./configure` (by default both
 `sandstorm`).  The easiest way to do this is to run as root:
 
 ```
-sudo -u sandstorm -g sandstorm /path/to/tempest
+sudo -u sandstorm -g sandstorm ./_build/tempest
```

Comment:
Hm, I wonder if instead of doing this we should tell people to use the one that's installed system wide `/usr/local/bin`; this would make it harder to run a build that is out of sync with the other executables.

Reply:
Might be better, not sure. I went with _build because the sandstorm-import-tool is there, and we can presume we know the user is sitting in the repo folder at this point in the instructions, and I assume if doing dev work wants the latest dev build. But if it depends on files installed elsewhere anyways, you are probably right?